sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/EventProcessor.cs

```diff
@@ -14,12 +15,10 @@ namespace Azure.Messaging.EventHubs.Processor
 {
     /// <summary>
     ///   Receives <see cref="EventData" /> as they are available for a partition, in the context of a consumer group, and routes
-    ///   them to a partition processor instance to be processed.
+    ///   them to be processed by a function provided by the user.
```

Comment:
nit:  Maybe something like "Consumes events for the configured Event Hub and consumer group across all partitions, making them available for processing through the provided handlers."    ?